addons/mail/models/mail_composer_mixin.py

```diff
@@ -66,8 +66,13 @@ def _compute_body_has_template_value(self):
         for composer_mixin in self:
             if not tools.is_html_empty(composer_mixin.body) and composer_mixin.template_id:
                 template_value = composer_mixin.template_id.body_html
-                sanitized_template_value = tools.html_sanitize(template_value)
-                composer_mixin.body_has_template_value = composer_mixin.body in (template_value, sanitized_template_value)
+                # matching email_outgoing sanitize level
+                sanitize_vals = {'silent': True, 'sanitize_tags': False, 'sanitize_attributes': False,
+                'sanitize_style': True, 'sanitize_form': True, 'sanitize_conditional_comments': False,
+                'output_method': 'xml', 'strip_style': False, 'strip_classes': False}
```

Comment:
Just to ease reading, and also ease future change (we can change a value, diff will be easier to read)
```suggestion
                sanitize_vals = {
                    'output_method': 'xml',
                    'sanitize_attributes': False,
                    'strip_classes': False,
                    'sanitize_conditional_comments': False,
                    'sanitize_form': True,
                    'sanitize_style': True, 
                    'sanitize_tags': False, 
                    'silent': True,
                    'strip_style': False,
                }
```